rosdep/python.yaml

```diff
@@ -9238,7 +9244,9 @@ python3-pyswarms-pip:
     pip:
       packages: [pyswarms]
 python3-pytest:
-  alpine: [py3-pytest]
+  alpine:
+    '*': [py3-pytest7]
```

Comment:
The version of [pytest for Ubuntu 24.04](https://launchpad.net/ubuntu/noble/+package/python3-pytest) (ROS 2 jazzy) is 7 but pytest for Alpine 3.20 is [8](https://pkgs.alpinelinux.org/package/v3.20/main/x86_64/py3-pytest). pytest 7 and 8 are [not compatible](https://github.com/pytest-dev/pytest/issues/11868) so py3-pytest7 should be used for Alpine 3.20.